index.bs

```diff
@@ -1219,7 +1219,7 @@ following Web IDL.
     dictionary CollectedClientData {
         required DOMString           challenge;
         required DOMString           origin;
-        required DOMString           hashAlg;
+        required DOMString           hashAlgorithm;
```

Comment:
AFAICT, the short form, "alg", is used in all the data structures handled by authenticators. 

improved egrep for "alg":
```
> grep -niE "alg[:;\s}\.| ]+" index.bs
579:    1. Let |normalizedAlgorithm| be the result of [=normalizing an algorithm=] [[!WebCryptoAPI]], with |alg| set to
580:        <code>|current|.{{PublicKeyCredentialParameters/alg}}</code> and |op| set to `"generateKey"`. If an error occurs during
989:        required AlgorithmIdentifier          alg;
1711:            ; All public key formats must include an alg name
1712:            pubKeyTemplate = { alg: text }
1716:            rsaPubKey = { alg: rsaAlgName, n: biguint, e: uint }
1720:            eccPubKey = { alg: eccAlgName, x: biguint, y: biguint }
2075:                           alg: rsaAlgName / eccAlgName,
2080:                           alg: "ED256" / "ED512",
2114:    mechanism. It sets |x5c| to the certificate chain of the attestation public key and |alg| to the algorithm of the
2119:    through an authenticator-specific mechanism (see [[!FIDOEcdaaAlgorithm]]). It sets |alg| to the algorithm of the
2123:    and signing the result using the credential private key. It sets |alg| to the algorithm of the credential private key, and
2134:        attestation public key in |x5c| with the algorithm specified in |alg|.
2146:    - Validate that |alg| matches the algorithm of the credential private key in |authenticatorData|.
2148:        credential public key with |alg|.
2200:                            alg: rsaAlgName / eccAlgName,
2204:                            alg:  "ED256" / "ED512",
2273:        algorithm specified in |alg|.
2932:        ...                                         -- all public key alg etc.
2982:        ...                                         -- all public key alg etc.
3056:        ...                    -- all public key alg etc.
```